alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultAuditAccessor.java

```diff
@@ -261,12 +261,12 @@ public AuditEntryModel convertToAuditEntryModelFromNotification(AlertNotificatio
             String errorMessage = auditEntryEntity.getErrorMessage();
             String errorStackTrace = auditEntryEntity.getErrorStackTrace();
 
-            Optional<ConfigurationJobModel> commonConfig = jobAccessor.getJobById(jobId);
+            Optional<DistributionJobModel> distributionJobModel = jobAccessor.getJobById(jobId);
             String distributionConfigName = null;
             String eventType = null;
-            if (commonConfig.isPresent()) {
-                distributionConfigName = commonConfig.get().getName();
-                eventType = commonConfig.get().getChannelName();
+            if (distributionJobModel.isPresent()) {
+                distributionConfigName = distributionJobModel.get().getName();
+                eventType = distributionJobModel.get().getChannelDescriptorName();
```

Comment:
You may want to call `distributionJobModel.get()` once, rather than pulling out the DistributionJobModel from the optional on both lines.

Reply:
I changed it to 
```
String distributionConfigName = distributionJobModel.map(DistributionJobModelData::getName).orElse(null);
            String eventType = distributionJobModel.map(DistributionJobModelData::getChannelDescriptorName).orElse(null);
```